src/smc-webapp/project_store.coffee

```diff
@@ -357,14 +357,21 @@ class ProjectActions extends Actions
             foreground_project : true
             chat               : undefined
             chat_width         : undefined
+            ignore_kiosk       : false
             new_browser_window : false     # open in entirely new browser window with a new random session.
 
+        #if DEBUG then console.log("ProjectStore::open_file: #{misc.to_json(opts)}")
+
+        # intercept any requests if in kiosk mode
+        if (not opts.ignore_kiosk) and (redux.getStore('page').get('fullscreen') == 'kiosk')
+            return
+
         if opts.new_browser_window
             # options other than path don't do anything yet.
             url  = (window.app_base_url ? '') + @_url_in_project('files/' + opts.path)
             url += '?session=' + misc.uuid().slice(0,8)
             url += '&fullscreen=default'
-            window.open(url, '_blank', 'height=640,width=800')
+            misc_page.open_new_tab(url, true)
```

Comment:
Revert this or improve open_new_tab to have more options.  The new_browser_windo thing is used for implementing the shift+click on a tab to create a popout window.   Read the code of `misc_page.open_new_tab(url, true)` -- it's different than ` window.open(url, '_blank', 'height=640,width=800')`, in that it doesn't have the height/width params, which I think make the window appear as a new pop-up rather than new tab.